core/src/modules/IStructureModule.cpp

```diff
@@ -7,27 +7,24 @@
 
 #include "include/IStructureModule.hpp"
 
-#include "include/DevGrid.hpp"
 #include "include/ParametricGrid.hpp"
 #include "include/RectangularGrid.hpp"
 
 namespace Module {
 
-const std::string DEV_GRID = "DevGrid";
 const std::string RECT_GRID = "RectangularGrid";
 const std::string PARAMETRICGRID = "ParametricGrid";
 
 template <>
 Module<Nextsim::IStructure>::map Module<Nextsim::IStructure>::functionMap = {
-    { DEV_GRID, newImpl<Nextsim::IStructure, Nextsim::DevGrid> },
     { RECT_GRID, newImpl<Nextsim::IStructure, Nextsim::RectangularGrid> },
     { PARAMETRICGRID, newImpl<Nextsim::IStructure, Nextsim::ParametricGrid> },
 };
 template <>
-Module<Nextsim::IStructure>::fn Module<Nextsim::IStructure>::spf = functionMap.at(DEV_GRID);
+Module<Nextsim::IStructure>::fn Module<Nextsim::IStructure>::spf = functionMap.at(RECT_GRID);
```

Comment:
@einola @timspainNERSC I am not sure about this part -- can it go, or should DEV_GRID be replaced by RECT_GRID? Is it needed somewhere?

Reply:
It should be replaced by `RECT_GRID`. This line designates the default implementation for this Module.